.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
We can add this, but I would recommend setting up a global gitignore file instead. Something like [this](https://github.com/leighmcculloch/dotfiles/blob/master/files/gitignore_global) is relatively common.

You can have your global gitignore file live anywhere on your system, I have mine at `~/.gitignore_global`, and you tell git where to find it with:

```
git config --global core.excludesfile '~/.gitconfig'
```

That command will edit your `~/.gitconfig` file adding a line like this:
https://github.com/leighmcculloch/dotfiles/blob/93aaaf4/files/gitconfig#L4

Reply:
oooh TIL! 💯  will set this up